src/Componenets/Categories.jsx

```diff
@@ -15,7 +15,7 @@ const Categories = () => {
   }, []);
```

Comment:
This is working but I'd recommend using axios for your API requests, rather than fetch